requirements/common_constraints.txt

```diff
@@ -0,0 +1,28 @@
+# A central location for most common version constraints
```

Comment:
I’m still not clear on our process, but couldn’t you update this comment with a link to the real common constraints, and an explanation for having a (what I think is temporary) local copy?